### DISCUSSION TITLE
How to get the font size and font family of the current selected text in TipTap?

### DISCUSSION THREAD

**Discussion by THPubs** [ueberdosis/tiptap#5236] [General Questions]
In TipTap how to get the attributes like font size and font family onSelectionUpdate? I'm trying to set the selected font in the font selection dropdown.

**Comment by deki23** [comment]
@THPubs 
How did you manage to do it?

**Comment by martinmolema** [comment]
I have an array of fonts (listOfFontFamilies) that I check against the selection :

this.currentFontFamily = this.listOfFontFamilies.find(font => this.editor.isActive('textStyle', {fontFamily: font})) || '';


Also see the warning (https://tiptap.dev/docs/editor/extensions/functionality/fontfamily ) : Be aware that editor.isActive('textStyle', { fontFamily: 'Font Family' }) will return the font family as set by the [browser's CSS rules](https://developer.mozilla.org/en-US/docs/Web/CSS/font-family#family-name) and not as you would have expected when setting the font family.
